Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

Changes CA directory in the sample configuration file.

--- tools/stunnel.conf-sample.in	2016-07-05 14:27:57.000000000 -0700
+++ tools/stunnel.conf-sample.in	2017-03-21 02:49:10.831006080 -0700
@@ -45,10 +45,10 @@
 
 ; ***************************************** Example TLS client mode services
 
-; The following examples use /etc/ssl/certs, which is the common location
-; of a hashed directory containing trusted CA certificates.  This is not
-; a hardcoded path of the stunnel package, as it is not related to the
-; stunnel configuration in @sysconfdir@/stunnel/.
+; The following examples use /etc/openssl/certs, which is the location
+; of directory containing hashed links to trusted CA certificate on Solaris.
+; This is not a hardcoded path of the stunnel package, as it is not related
+; to the stunnel configuration in @sysconfdir@/stunnel/.
 
 [gmail-pop3]
 client = yes
@@ -55,7 +55,7 @@
 accept = 127.0.0.1:110
 connect = pop.gmail.com:995
 verifyChain = yes
-CApath = /etc/ssl/certs
+CApath = /etc/openssl/certs
 checkHost = pop.gmail.com
 OCSPaia = yes
 
@@ -64,7 +64,7 @@
 accept = 127.0.0.1:143
 connect = imap.gmail.com:993
 verifyChain = yes
-CApath = /etc/ssl/certs
+CApath = /etc/openssl/certs
 checkHost = imap.gmail.com
 OCSPaia = yes
 
@@ -73,7 +73,7 @@
 accept = 127.0.0.1:25
 connect = smtp.gmail.com:465
 verifyChain = yes
-CApath = /etc/ssl/certs
+CApath = /etc/openssl/certs
 checkHost = smtp.gmail.com
 OCSPaia = yes
 
